openapi-generator/lib/src/openapi_generator_runner.dart

```diff
@@ -30,9 +30,7 @@ class OpenapiGenerator extends GeneratorForAnnotation<annots.Openapi> {
       communication: OutputMessage(
         message: [
           '',
-          ':::::::::::::::::::::::::::::::::::::::::::',
-          '::      Openapi generator for dart       ::',
-          ':::::::::::::::::::::::::::::::::::::::::::',
+          '     Openapi generator for dart      ',
```

Comment:
Please let's retain the previous format.

The way in this PR does not produce the same effect

<img width="657" alt="Screenshot 2023-08-28 at 19 52 46" src="https://github.com/gibahjoe/openapi-generator-dart/assets/9147147/c4ed3935-b750-4c3d-aa9a-2a95864f5878">